templates/site/settings/modal-inactivewarning-email.html

```diff
@@ -0,0 +1,99 @@
+              </td>
+              <td>
+                Fullname of user submitted the idea
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <b> URL </b>
+              </td>
+              <td>
+                The url of the website the idea was submitted to
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <b> HOST </b>
+              </td>
+              <td>
+                The website, without www, the which idea was submitted to.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <b> idea.id </b>
+              </td>
+              <td>
+                The id of a submitted idea
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <b> idea.title </b>
+              </td>
+              <td>
+                The title of a submitted idea
+              </td>
+            </tr>
+
+          </table>
+          <br />
+          <p>
+            This is an example template:
+          </p>
+          <textarea class="form-control" style="min-height: 400px;">{% raw %}<h4> Beste  {{user.fullName | default('indiener')}},</h4>
+            <br/>
+            <br/>
+            Bedankt voor jouw inzending.
+            <br/>
+            Deze is goed ontvangen.
+            <br/>
+            <br/>
+            <p >
+	            Hartelijke groet,<br>
+	            Team {{SITENAME}}
+            </p>{% endraw %}</textarea>
+          <br />
+          <h3> How to link to the page of submitted idea? </h3>
+          You can add a link to your website on the page where submissions are shown and add the ID to it.
+          For instance for westbegroot.nl has the plan page for submission so a link to the page of a submitted idea is structured as follows:
+          https://www.westbegroot.nl/plan/10
+          This a working example for Westbegroot.nl:
+          <textarea class="form-control" style="min-height: 150px;">{% raw %}<a href="https://www.westbegroot.nl/plan/{{idea.id}}">
+            {{idea.title}}
+          </a>{% endraw %}</textarea>
+          <br />
+          <h3> How to add an image? </h3>
+          If you upload an image to the CMS and get the url. For instance header image:
+          https://westbegroot.amsterdam.nl/uploads/attachments/ck07xgjwz00pa2u3w6yvaclno-wb2020-headerfoto-2x-v2.full.png
+          Make sure image are not too big, preferably around twice the size it's displayed in the email.
+          Setting width="560" will make the image stretch full width for old email clients, newer ones support 100% which makes the image responsive.
+
+          <textarea class="form-control" style="min-height: 200px;">{% raw %}<img src="https://westbegroot.amsterdam.nl/uploads/attachments/ck07xgjwz00pa2u3w6yvaclno-wb2020-headerfoto-2x-v2.full.png" width="560" style="display:block;width:100%">{% endraw %}</textarea>
```

Comment:
It would be great to have a correct example (can be the default template — really basic) with available variables here

Reply:
I do agree. I do expect the a real template to be added before we go live, and will then add that as the default here.